be/src/exec/json_scanner.h

```diff
@@ -68,13 +68,13 @@ class JsonScanner : public BaseScanner {
     // Close this scanner
     void close() override;
 
-private:
+protected:
     Status open_file_reader();
     Status open_line_reader();
     Status open_json_reader();
     Status open_next_reader();
 
-private:
+protected:
```

Comment:
Maybe we can delete some redundant protected

Reply:
I will re-check it.

Reply:
> Maybe we can delete some redundant protected

In order to distinguish the function and variables , I prefer to use 2 'protected' key words.